tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs

```diff
@@ -465,19 +465,58 @@ public static Result IsProcessV1(this MockSpan span) => Result.FromSpan(span)
                 .IsOptional("cmd.environment_variables")
                 .Matches("span.kind", "internal"));
 
-        public static Result IsRabbitMQV1(this MockSpan span) => Result.FromSpan(span)
+        public static Result IsRabbitMQAdminV1(this MockSpan span) => Result.FromSpan(span)
+            .WithMarkdownSection("Rabbit - Admin")
             .Properties(s => s
                 .Matches(Name, "amqp.command")
                 .Matches(Type, "queue"))
             .Tags(s => s
                 .IsPresent("amqp.command")
+                .IsPresent("out.host")
                 .IsOptional("amqp.delivery_mode")
                 .IsOptional("amqp.exchange")
                 .IsOptional("amqp.routing_key")
                 .IsOptional("amqp.queue")
                 .IsOptional("message.size")
+                .IsPresent("peer.service")
+                .IsOptional("peer.service.remapped_from")
+                .MatchesOneOf("_dd.peer.service.source", "out.host", "peer.service")
                 .Matches("component", "RabbitMQ")
-                .IsPresent("span.kind"));
+                .Matches("span.kind", "client"));
+
+        public static Result IsRabbitMQInboundV1(this MockSpan span) => Result.FromSpan(span)
+            .WithMarkdownSection("Rabbit - Inbound")
+            .Properties(s => s
+                .Matches(Name, "amqp.process")
+                .Matches(Type, "queue"))
+            .Tags(s => s
+                .IsPresent("amqp.command")
+                .IsOptional("amqp.delivery_mode")
+                .IsOptional("amqp.exchange")
+                .IsOptional("amqp.routing_key")
+                .IsOptional("amqp.queue")
+                .IsOptional("message.size")
+                .Matches("component", "RabbitMQ")
+                .Matches("span.kind", "consumer"));
```

Comment:
It looks like #4398 just got merged, which is introducing some merge conflicts with the snapshots. Sorry that this stepped on your PR, I didn't catch it 😞 

Now that that's merged, after updating to master can you update tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/BasicConsumeIntegration.cs so that is uses `span.kind=consumer`?